src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -6943,10 +6939,8 @@ bool ClrDataAccess::TargetConsistencyAssertsEnabled()
     return m_fEnableTargetConsistencyAsserts;
 }
 
-#ifdef FEATURE_CORESYSTEM
 #define ctime_s _ctime32_s
```

Comment:
Is this needed? I would expect `ctime_s` to come from standard CRT headers now. (The original port to CoreSystem had old minimal CRT that was missing number of `*_s` overloads.)